mlflow/sagemaker/container/__init__.py

```diff
@@ -66,6 +66,9 @@ def _serve():
         print("activating custom environment")
         env = conf[pyfunc.ENV]
         env_path_dst = os.path.join("/opt/mlflow/", env)
+        env_path_dst_dir = os.path.dirname(env_path_dst)
```

Comment:
@tomasatdatabricks do you know why this might have been added?

Reply:
Because we were attempting to copy files to a destination directory that may not exist; ran into an error during copying due to this. There may be an easier way to force directory creation by adding a flag to the copy command.

Reply:
Yes, it looks reasonable to me.  Btw, I fix this in my pending PR too so we might need to merge it at some point.